Graphs/SimpleTextGraph/index.js

```diff
@@ -48,8 +48,7 @@ class SimpleTextGraph extends AbstractGraph {
         } = this.props;
 
         const {
-            innerWidth,
-            innerHeight,
+            dimension,
```

Comment:
this is new have you checked the vis-config for the configuration of innerHeight/innerWidth since there are removed now? Thanks